docs/site/Life-cycle-observer-generator.md

```diff
@@ -0,0 +1,83 @@
+---
+lang: en
+title: 'Life cycle observer generator'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Life-cycle-observer-generator.html
+---
+
+{% include content/generator-create-app.html lang=page.lang %}
+
+### Synopsis
+
+Adds a new [LifeCycleObserver](Life-cycle.md) class to a LoopBack application.
+
+```sh
+lb4 observer [--group <group>] [<name>]
```

Comment:
Please describe the new generator in CLI README too, see https://github.com/strongloop/loopback-next/blob/master/packages/cli/README.md

Reply:
Good catch.